components/search_engines/brave_prepopulated_engines.cc

```diff
@@ -122,11 +122,11 @@ const PrepopulatedEngine qwant = {
 };
 
 const PrepopulatedEngine startpage = {
-    L"StartPage",
+    L"Startpage",
     L":sp",
     "https://www.startpage.com/favicon.ico",
     "https://www.startpage.com/do/"
-    "dsearch?query={searchTerms}&cat=web&pl=opensearch",
+    "search?q={query}&segment=startpage.brave",
```

Comment:
Need to bump `kBraveCurrentDataVersion` at the top of the file?

Reply:
yes- you're absolutely right! Thanks for catching 😄

Reply:
Fixed! 😄